osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -528,6 +528,19 @@ protected HitObjectComposer(Ruleset ruleset)
 
         public virtual string ConvertSelectionToString() => string.Empty;
 
+        /// <summary>
+        /// The custom logic that decides whether a HitObject should be selected when clicking an editor timestamp link
+        /// </summary>
+        /// <param name="hitObject">The hitObject being checked</param>
+        /// <param name="objectInfo">A single hitObject's information created with <see cref="ConvertSelectionToString"/></param>
+        /// <returns>Whether a HitObject should be selected or not</returns>
+        public virtual bool HandleHitObjectSelection(HitObject hitObject, string objectInfo) => false;
+
+        /// <summary>
+        /// A character that separates the selection in <see cref="ConvertSelectionToString"/>
+        /// </summary>
+        public virtual char ObjectSeparator => ',';
```

Comment:
This virtual member is much too specific and should not exist.

Reply:
Removed in [`4e1e197`](https://github.com/ppy/osu/pull/25371/commits/4e1e19728cb8ff9e11f18ab9c0e8635c2cc2ba9a)